DribbbleSDKExample/DribbbleSDKExample/MainViewController.m

```diff
@@ -94,15 +96,73 @@ - (void)setupApiClient {
         if (error.domain == NSURLErrorDomain && ![weakSelf.apiClient isUserAuthorized]) {
             [weakSelf performSegueWithIdentifier:kSegueIdentifierAuthorize sender:nil];
         } else {
-            [UIAlertView bk_showAlertViewWithTitle:@"Error" message:[error localizedDescription] cancelButtonTitle:@"OK" otherButtonTitles:nil handler:nil];
+            if (error.code != 404) {
+                [UIAlertView bk_showAlertViewWithTitle:@"Error" message:[error localizedDescription] cancelButtonTitle:@"OK" otherButtonTitles:nil handler:nil];
+            }
         }
     };
+    
+    
+    if (!self.delegate.user) {
+        [self loadMockData];
+    }
+    
+    self.apiCallWrappers = [ApiCallFactory demoApiCallWrappers];
+    [self.tableView reloadData];
+}
+
+- (void)loadMockData {
+    [self.apiClient loadUserInfoWithResponseHandler:^(DRApiResponse *response) {
+        if (response.object) {
+            self.delegate.user = response.object;
+        }
+        
+        [self.apiClient loadShotsWithUser:self.delegate.user.userId params:@{} responseHandler:^(DRApiResponse *response) {
+            if (response.object) {
+                if ([response.object isKindOfClass:[NSArray class]]) {
+                    if ([response.object count]) {
+                        DRShot *shot = [response.object firstObject];
+                        if(shot) {
+                            self.delegate.shot = shot;
+                            [self.apiClient loadCommentsWithShot:shot.shotId params:@{} responseHandler:^(DRApiResponse *response) {
+                                if (response.object) {
+                                    for (DRComment *comment in response.object) {
+                                        if (([comment.body isEqualToString:@"<p>API test updated comment</p>"] || [comment.body isEqualToString:@"<p>API test comment</p>"]) &&
+                                            comment.user.userId == self.delegate.user.userId) {
+                                            self.delegate.comment = comment;
+                                            self.apiCallWrappers = [ApiCallFactory demoApiCallWrappers];
+                                            [self.tableView reloadData];
+                                        }
+                                    }
+                                    [self.apiClient loadAttachmentsWithShot:shot.shotId params:@{} responseHandler:^(DRApiResponse *response) {
+                                        if (response.object && [response.object isKindOfClass:[NSArray class]]) {
+                                            DRShotAttachment *attachment = [response.object lastObject];
```

Comment:
please use the same getter for arrays with 1 object. preferred it the "firstObject"